frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -183,7 +262,7 @@ const HostSummary = ({
               `${titleData.os_version}`
             ) : (
               <Button
-                onClick={() => toggleOSPolicyModal && toggleOSPolicyModal()}
```

Comment:
Looks like this could still be undefined based on the `IHostSummaryProps`? I wonder if we should try to tighten up that interface definition.

Fun fact, optional chaining works with functions: `toggleOSPolicyModal?.()`

Reply:
Ah thanks

Reply:
that's a funny syntax

Reply:
https://www.youtube.com/watch?v=sRWE5tnaxlI

Reply:
\** backs away slowly **

<img width="846" alt="Screenshot 2023-02-14 at 12 16 05 PM" src="https://user-images.githubusercontent.com/61553566/218851957-7688bc11-9d2b-409b-924f-10edb1d765c0.png">

Reply:
I love his YT channel so much